.github/workflows/docker.yml

```
@@ -499,6 +500,18 @@ jobs:
 
+      # Smoke-test the just-built backend image at PR time so a broken
+      # boot path (404 on the wrong probe URL, missing migration,
+      # entrypoint crash, etc.) fails the originating PR with a red
+      # ❌ on the commit row instead of the finalize-release step
+      # after a tag has already been cut. The image is loaded into
+      # the local Docker daemon by build-scan-image; we run it here
+      # in the same job. Probes /api/v1/healthz with a 120s deadline.
+      - name: Smoke-test backend image
+        uses: ./.github/actions/smoke-test-backend-image
+        with:
+          image: ${{ steps.build_scan.outputs.ref_amd64 }}
```

actionlint (1.7.12)

[error] property "ref_amd64" is not defined in object type {}

(expression)